src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -187,6 +187,67 @@ def test_aks_create_service_no_wait(self, resource_group, resource_group_locatio
         # show again and expect failure
         self.cmd('aks show -g {resource_group} -n {name}', expect_failure=True)
 
+    @ResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
```

Comment:
What's the necessity of specifying the `name_prefix` and `random_name_length` here?

Reply:
This is for creation of random group name prefix makes sure that multiple module tests dont overlap each other and random length makes sure that we can run multiple tests in parallel.

Reply:
Also just reused the previous tests to keep the styling intact.

Reply:
I understand the randomness to avoid duplication part (I wrote the code behind this). Let me phrase my question in this way: if you don't provide the `name_prefix` and `random_name_length` will the test run successfully? After all, both parameters have their default setting, which you don't have to specifically overwrite.

Reply:
Default settings are name_prefix='clitest.rg' and random_name_length=75, i tested the basic unitest and unfortunately aks rp doesnt like the "." in the resource group name and/or the length

msrestazure.azure_exceptions.CloudError: 400 Client Error: Bad Request for url: https://management.azure.com/subscriptions/5eed6188-c6d6-4966-9543-28b3c88ee4e1/resourceGroups/clitest.rgjug4amagf7n42kaltr3tbreekmwv2gfd76u6w3swsqsiphvtfhc23le2kq7iihc36/providers/Microsoft.ContainerService/managedClusters/cliakstestjwiv6f?api-version=2018-03-31

huh! seems like the limitation should have been recorded. Will log an issue on this.

Reply:
Logged issue https://github.com/Azure/azure-cli/issues/7389

Reply:
Ok. That's the answer I'm looking for. Thank you.